timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -71,9 +71,9 @@ NonbondedAllPairs<RealType>::NonbondedAllPairs(
     cudaSafeMalloc(&d_rebuild_nblist_, 1 * sizeof(*d_rebuild_nblist_));
     gpuErrchk(cudaMallocHost(&p_rebuild_nblist_, 1 * sizeof(*p_rebuild_nblist_)));
 
-    cudaSafeMalloc(&d_sort_keys_in_, N_ * sizeof(d_sort_keys_in_));
-    cudaSafeMalloc(&d_sort_keys_out_, N_ * sizeof(d_sort_keys_out_));
-    cudaSafeMalloc(&d_sort_vals_in_, N_ * sizeof(d_sort_vals_in_));
+    cudaSafeMalloc(&d_sort_keys_in_, N_ * sizeof(*d_sort_keys_in_));
```

Comment:
With the previous commit, found a couple places where we were allocating more memory than we needed

Reply:
Good call